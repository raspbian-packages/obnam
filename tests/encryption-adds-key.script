#!/bin/sh
# Copyright 2011  Lars Wirzenius
# 
# This program is free software: you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation, either version 3 of the License, or
# (at your option) any later version.
# 
# This program is distributed in the hope that it will be useful,
# but WITHOUT ANY WARRANTY; without even the implied warranty of
# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
# GNU General Public License for more details.
# 
# You should have received a copy of the GNU General Public License
# along with this program.  If not, see <http://www.gnu.org/licenses/>.

set -e

gpgkey='3B1802F81B321347'
gpgkey2='DF3D13AA11E69900'
$SRCDIR/tests/backup --encrypt-with="$gpgkey"
$SRCDIR/tests/obnam --encrypt-with="$gpgkey" add-key --keyid="$gpgkey2"
$SRCDIR/tests/obnam --encrypt-with="$gpgkey" list-keys | grep '^key:'

