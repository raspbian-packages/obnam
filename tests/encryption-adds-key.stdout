key: DF3D13AA11E69900
key: 3B1802F81B321347
